caiman/base/movies.py

```diff
@@ -1157,7 +1158,7 @@ def load(file_name,fr=30,start_time=0,meta_data=None,subindices=None,shape=None,
     if os.path.exists(file_name):
         _, extension = os.path.splitext(file_name)[:2]
 
-        if extension == '.tif' or extension == '.tiff':  # load avi file
+        if extension == '.tif' or extension == '.tiff' or extension == '.TIF':  # load avi file
```

Comment:
We should probably just lowercase the extension right after splitting it off from the file before comparison, so we don't need to introduce case variants for all the extensions.

Reply:
Agree. I looked for the appearances of the word `extension` and switched it to always be lowercase. Hopefully that should capture everything